eng/containers/UbuntuNetCoreKeyring/Dockerfile

```diff
@@ -1,24 +1,53 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-focal AS build
+FROM mcr.microsoft.com/powershell:7.1.4-ubuntu-focal AS build
 
 ENV \
     NO_AT_BRIDGE=1 \
     DOCKER_CONTAINER_NAME="ubuntu_netcore_keyring" \
-    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
-    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetCoreSDK-Ubuntu-20.04
+     # Unset ASPNETCORE_URLS from aspnet base image
+    ASPNETCORE_URLS= \
+    # Do not generate certificate
+    DOTNET_GENERATE_ASPNET_CERTIFICATE=false \
+    # SDK version
+    DOTNET_SDK_VERSION_5.0=5.0.401 \
+    DOTNET_SDK_VERSION_3_1=3.1.413 \
+    # Enable correct mode for dotnet watch (only mode supported in a container)
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    # Skip extraction of XML docs - generally not useful within an image/container - helps performance
+    NUGET_XMLDOC_MODE=skip \
+    # PowerShell telemetry for docker image usage
+    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetSDK-Ubuntu-20.04 \
+    # Setup Dotnet envs
+    DOTNET_ROOT=/usr/share/dotnet  \
+    PATH=$PATH:usr/share/dotnet    
 
 # Install GNOME keyring
 RUN apt-get update \
     && apt-get install -y \
         libsecret-1-dev \
         dbus-x11 \
         gnome-keyring \
-        python
+        python \
+        curl
 
-# Install PowerShell && .net core runtime 2.1
-RUN wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb \
-    && dpkg -i packages-microsoft-prod.deb \
-    && apt-get update \
-    && apt-get install -y powershell \
-    && pwsh --version \
-    && apt-get install -y dotnet-runtime-2.1 \
-    && dotnet --list-runtimes
+# Below taken from https://hub.docker.com/_/microsoft-dotnet-sdk
+# 5.0.401-focal-amd64, 5.0-focal-amd64, 5.0.401-focal, 5.0-focal	Dockerfile	Ubuntu 20.04	09/14/2021
```

Comment:
When I originally went here I wasn't sure what I was looking for. It might be worth also adding this link `https://github.com/dotnet/dotnet-docker/blob/ca43690e6e4ac9c21a990a012dc2eb07e78daf08/src/sdk/5.0/focal/amd64/Dockerfile`.